### PR TITLE
Add blog post about navigation and profile page

### PR DESCRIPTION
**Pull request by dmarcoux**
Preview: https://deploy-preview-272--openbuildservice.netlify.app/2020/10/22/navigation-and-profile-page/

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit d0929d49f65dc207a0e2f9f16be4e541a8441995

https://deploy-preview-272--openbuildservice.netlify.app

File: _posts/development/2020-10-20-navigation-and-profile-page.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Navigation and Profile Page Redesign
+category: development
+---
+
+On our quest to improve usability, we've been tuning various parts of OBS. This
+time, we've worked on the navigation and redesigned the profile page. All of this
+is available in the beta program, so be sure to [join](https://openbuildservice.org/2018/10/04/the-beta-program/)
+if you haven't done so already!
```
Comment (saraycp) [inline _posts/development/2020-10-20-navigation-and-profile-page.md:10]: For me, `yet` sounds better than `already`, but feel free to leave it like it is if you are sure it's correct.

File: _posts/development/2020-10-20-navigation-and-profile-page.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+this new design, the page looks much nicer on all devices, especially on those
+with smaller screens.
+
+Ever wondered how a user is involved in projects or packages? This is a lot
+easier to find out now with the reworked `Involved Projects/Packages` tab.
+Simply use the `Role` filter and voilà, you know if they are the maintainer,
+reviewer or more.
+
+In addition to your avatar, the new biography field allows you to personalize
+your profile page with a short text. Be creative, the sky is the limit!
```
Comment (saraycp) [inline _posts/development/2020-10-20-navigation-and-profile-page.md:34]: The sky and 250 characters for now ;)

**Review by saraycp — COMMENTED** [review]
I like it! Only a few details that I commented.
As soon as we have the screenshots, I'll approve it.

File: _posts/development/2020-10-20-navigation-and-profile-page.md (hunk 2 above)
Comment (dmarcoux) [reply _posts/development/2020-10-20-navigation-and-profile-page.md:34]: I don't want to destroy their dreams.... hahaha

File: _posts/development/2020-10-20-navigation-and-profile-page.md (hunk 1 above)
Comment (dmarcoux) [reply _posts/development/2020-10-20-navigation-and-profile-page.md:10]: They are both valid and only slightly differ. I hope that people already joined the beta program, so I prefer to use `already`. See below for details on the difference between `already` and `yet`.

> Already refers to things which have happened or which people think may have happened. Yet refers to things which have not happened or which people think may not have happened.

_from [`Already, still or yet?` from Cambridge Dictionary
](https://dictionary.cambridge.org/grammar/british-grammar/already-still-or-yet)_

**Review by krauselukas — COMMENTED** [review]
@dmarcoux nice blog post!

**Review by danidoni — APPROVED** [review]

File: _posts/development/2020-10-20-navigation-and-profile-page.md (hunk 1 above)
Comment (saraycp) [reply _posts/development/2020-10-20-navigation-and-profile-page.md:10]: great! :+1:

File: _posts/development/2020-10-20-navigation-and-profile-page.md (hunk 2 above)
Comment (saraycp) [reply _posts/development/2020-10-20-navigation-and-profile-page.md:34]: :smile:

**Review by eduardoj — APPROVED** [review]

**Comment by dmarcoux** [comment]
Still need to add a screenshot/GIF of the profile page.

**Comment by dmarcoux** [comment]
I've added a GIF of the profile page on mobile.

**Review by eduardoj — APPROVED** [review]

**Pull request merged**